src/wizard/logic-app/GenerateBuildDefinitionStep.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Workflow } from "azure-arm-logic/lib/models";
+import * as fse from "fs-extra";
+import * as glob from "glob";
+import * as path from "path";
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { generateTemplateParameter, generateTemplateParameterDefinition, generateTemplateResource } from "../../utils/logic-app/templateUtils";
+import { IBuildDefinitionWizardContext } from "./createBuildDefinition";
+
+export class GenerateBuildDefinitionStep extends AzureWizardExecuteStep<IBuildDefinitionWizardContext> {
+    public async execute(wizardContext: IBuildDefinitionWizardContext): Promise<IBuildDefinitionWizardContext> {
+        const { location, templateParameterDefinitions, templateParameters, templateResources, workspaceFolderPath } = wizardContext;
+
+        const definitions = await new Promise<string[]>((resolve, reject) => {
+            glob(path.join(workspaceFolderPath!, "**/*.definition.json"), (err, matches) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    resolve(matches);
+                }
+            });
+        });
+        const names = definitions.map((definition) => path.basename(definition).replace(/\.definition\.json$/i, ""));
+
+        for (const name of names) {
+            const file = path.join(workspaceFolderPath!, `${name}.definition.json`);
+            const json = await fse.readJSON(file);
```

Comment:
I haven't tested this out and maybe I've just missed where you handle it. If I modify my JSON files and they aren't valid JSON anymore, what happens?

Reply:
It'll error out, e.g., "Unexpected end of JSON input," and terminate the process.  The wizard will catch the rejected promise and display the message.